package.json

```diff
@@ -124,6 +124,7 @@
     "test": "make test",
     "test-client": "NODE_ENV=test NODE_PATH=test:client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client test/runner.js",
+    "test-tests": "NODE_ENV=test NODE_PATH=test:test/test test/runner.js",
```

Comment:
It's minor, but the naming stands out as a bit inconsistent with the other two script names:

- `client` -> `test-client`
- `server` -> `test-server`
- `test` -> `test-test`__s__